### PR TITLE
[docs] Update branch name and tags

### PR DESCRIPTION
**Pull request by cherniavskii**
Preparation for the v7 release.
Covers https://www.notion.so/mui-org/v7-stable-release-9f3fded55b9c425c80c03e9813dc8171?pvs=4#8c61e94774ff4a6c8823490c0a11ec1d

### CODE REVIEW

**Review by LukasTy — APPROVED** [review]
🚀

File: docs/data/introduction/support/support.md [hunk 1]
```diff
@@ -48,8 +48,10 @@ You can browse the documentation, find an example close to your use case, and th
 #### Use starter templates
 
 You can use a starter template to build a reproduction case with:
+
 <!-- #default-branch-switch -->
```
[the remark this answers is not in the document]
Comment (cherniavskii) [reply docs/data/introduction/support/support.md:52]: We will switch it back to `next` when we start working on v8, right?

File: docs/data/migration/migration-charts-v6/migration-charts-v6.md [hunk 2]
```diff
@@ -4,6 +4,8 @@ productId: x-charts
 
 # Migration from v6 to v7
 
+<!-- #default-branch-switch -->
```
[the remark this answers is not in the document]
Comment (cherniavskii) [reply docs/data/migration/migration-charts-v6/migration-charts-v6.md:7]: Right, we won't need it here anymore, I'll remove it 👍🏻

File: docs/data/introduction/support/support.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (LukasTy) [reply docs/data/introduction/support/support.md:52]: Sorry, I was a bit "blind" and didn't notice the change in the link path. 🙈 👌

**Pull request merged**